tools/make/kube.mk

```diff
@@ -2,7 +2,7 @@
 ENVTEST_K8S_VERSION ?= 1.24.1
 # GATEWAY_API_VERSION refers to the version of Gateway API CRDs.
 # For more details, see https://gateway-api.sigs.k8s.io/guides/getting-started/#installing-gateway-api 
-GATEWAY_API_VERSION ?= $(shell go list -m -f '{{.Version}}' sigs.k8s.io/gateway-api)
+GATEWAY_API_VERSION ?= v0.5.1
```

Comment:
I am trying to figure out how to fetch the latest GWAPI manifests, since the v0.6.0 is not released yet.

Reply:
REsolved in https://github.com/envoyproxy/gateway/pull/778.